tox.ini

```diff
@@ -1,44 +1,42 @@
 [tox]
-envlist = py37,py38,py39,py310
+envlist = 
+    lint, 
+    format, 
+    flake8,
+    test
 
-[gh-actions]
-python =
-    3.7: py37
-    3.8: py38
-    3.9: py39
-    3.10: py310
-
-[testenv]
+[testenv:test]
 extras = test
-commands=
-    python -m pytest --cov --cov-report xml --cov-report term-missing --ignore=venv
+commands =
+    - python -m pytest --cov --cov-report xml --cov-report term-missing --ignore=venv
```

Comment:
this minus should not be here, otherwise this will always return success even if the tests fail